### PR TITLE
change default DynamicCamera constraints

### PR DESCRIPTION
**Pull request by LukasPaczos**
### Description

This PR changes the default `DynamicCamera` constraints:
- Tilt is now always 40 degrees instead of moving between 45 and 60 degrees. This significantly improves the performance and readability of the map.
- Zoom is now capped between 16.35 and 14  instead of 16 and 12. This offers a much better balance of overview and detail.

The PR also decreases the maximum zoom animation duration for the camera to decrease the chance of the animation being interrupted by the subsequent progress update.

### Changelog

```
<changelog>:warning: Changed `DynamicCamera`'s default tilt and zoom constraints to improve performance and readability of the map when in active navigation.</changelog>
```

### Screenshots or Gifs

The below simulations are sped up a little bit.

### Before

https://user-images.githubusercontent.com/16925074/114598833-a5750500-9c92-11eb-8de1-54821317e9d1.mp4

https://user-images.githubusercontent.com/16925074/114597776-6f835100-9c91-11eb-994f-8f79b4eb8523.mp4

### After

https://user-images.githubusercontent.com/16925074/114598859-ac037c80-9c92-11eb-8b11-e1a37830efea.mp4

https://user-images.githubusercontent.com/16925074/114597733-65f9e900-9c91-11eb-9a9c-ff3c5d9a534e.mp4

### CODE REVIEW

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/DynamicCamera.java [hunk 1]
```diff
@@ -42,20 +40,6 @@ public DynamicCamera(@NonNull MapboxMap mapboxMap) {
     this.mapboxMap = mapboxMap;
   }
 
-  @Override
-  public double tilt(@NonNull RouteInformation routeInformation) {
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/DynamicCamera.java:46]: Are we ok breaking SemVer here?

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/SimpleCamera.kt [hunk 2]
```diff
@@ -11,20 +11,20 @@ import com.mapbox.turf.TurfConstants
...
- * The default camera used by [MapboxNavigation].
+ * The base camera implementation.
  */
 open class SimpleCamera : Camera() {
 
     companion object {
         /**
          * Default tilt angle for the Camera
          */
-        protected const val DEFAULT_TILT = 50
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/SimpleCamera.kt:22]: Isn't this breaking SemVer?

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/DynamicCamera.java (hunk 1 above)
Comment (Guardiola31337) [reply libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/DynamicCamera.java:46]: BTW Metalava is ❌

Comment (LukasPaczos) [reply libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/DynamicCamera.java:46]: This isn't breaking semver, there's still the parent `SimpleCamera#tilt` available.

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/SimpleCamera.kt (hunk 2 above)
Comment (LukasPaczos) [reply libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/SimpleCamera.kt:22]: Same here and above - we're making a behavioral change that does impact the default experience, but we believe that the previous behavior was buggy and this is a clear improvement.

Comment (LukasPaczos) [reply libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/SimpleCamera.kt:22]: We should be very transparent in the release notes though.

**Review by Guardiola31337 — APPROVED** [review]

**Pull request merged**